src/schemas/utils.ts

```diff
@@ -170,34 +172,42 @@ export interface SubmissionEntity {
 }
 
 const convertClinicalSubmissionEntityToGql = (clinicalType: string, entity: SubmissionEntity) => {
+  const batchName = entity.batchName || undefined;
```

Comment:
This and other declarations can use optional chaining:
`const batchName = entity?.batchName;`

Reply:
done.